### ISSUE TITLE
Design Styleguide

### ISSUE THREAD

**Issue by himanshuc3** [publiclab/PublicLab.Editor#229]
### Please describe the problem (or idea)

> What happened just before the problem occurred? Or what problem could this idea solve?
Feature request.
Hi. I would like to design the styleguide for PublicLab.Editor. This way more time would be spent on solving functionality related problems rather than discussing on stylistic issues.

### What's your PublicLab.org username?
himichhabra14

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by gauravano** [comment]
cc @jywarren @publiclab/community-reps

**Comment by jywarren** [comment]
This is a good idea... we generally are using a lot from Bootstrap 3, but
I'm interested in seeing some ideas! https://getbootstrap.com/docs/3.3/

Would you like to do a small example of the kind of thing that'd go into a
style guide?

Thanks so much!!!! We definitely do need some design input!

**Comment by niravasher** [comment]
Hi! @jywarren , I would like to make the style guide as per discussed. We could use materialize.css as a change in place of bootstrap. There are also some javascript libraries that display content beautifully. Could you assign me an example regarding the same? I'd love to work upon it

**Comment by rexagod** [comment]
@niravasher This will be a major enhancement to say the least, and will require a very planned and detailed approach before making an implementation of any sort to avoid breaking changes. If you're proposing to shift the Editor's theme to materialize, can you first come up with some prototype screenshots of how it's going to look across different resolutions and devices? We can then proceed further with finalizing a design for the Editor. Thanks.

**Comment by niravasher** [comment]
You are right, shifting to materialize is likely to break functionality. The new UI looks something like this as referenced in issue #216 
![screenshot 8](https://user-images.githubusercontent.com/32176296/51740206-61998380-20b9-11e9-95ec-c15984e80904.png)
![screenshot 9](https://user-images.githubusercontent.com/32176296/51740216-678f6480-20b9-11e9-8a9b-a5b9ab170809.png)
![screenshot 10](https://user-images.githubusercontent.com/32176296/51740245-7e35bb80-20b9-11e9-8af1-4f8db96285a1.png)

**Comment by niravasher** [comment]
Oh absolutely, you are right @rexagod that will break functionality. I have opened up a discussion with the community if a new look could be implemented not changing any libraries.
